source/developers/how-tos/get-your-project-docs-on-rtd.rst

```diff
@@ -0,0 +1,71 @@
+How-to get your repository docs on docs.openedx.org
+###################################################
+
+.. Titles should be imperative. https://www.grammarly.com/blog/imperative-sentences/
```

Comment:
Is this link used?  My question/recommendation was going to be to have an example of an imperative title.

Reply:
This was just a comment, I've removed it.